### DISCUSSION TITLE
fight-compiler/borrowing

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#534] [Book Comments]
# fight-compiler/borrowing

Learn Rust with Example, Exercise and real Practice, written with ❤️ by https://course.rs team

https://practice.course.rs/fight-compiler/borrowing.html

**Comment by yomanthunder** [comment]
I am the first to get this far, lol (at least the first to comment and leave a reaction)! 😎🎉 I’ve learned a lot from Rust by practicing, but now I realize that it’s not enough—I need to read *The Rust Programming Language* book as well. 📖 There are important concepts to learn from there, too. After that, you’re good to go for Rustlings and more practice. 🚀 You could also try out CSES or Exercism for algorithm practice, but I’d prefer to work on some projects, as I’ve already learned most of the key concepts required for project work. 🔧💻 

👉 **Please leave a reaction on this message if you make it to the end!** 👍💬🔥

**Comment by iamtaojin** [comment]
Am I the second?

**Reply by yomanthunder** [reply]
Maybe 😂.
